Resources/test-parameters.json

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "functionAppPrefix": {
+        "value": "dfc-foo-prov-ref"
```

Comment:
Looks like this was copy and pasted from another project without updating to reflect the name of this project